### ISSUE TITLE
When trying the SSO flow , with console and my account having two language modes , the UI provides mix language.

### ISSUE THREAD

**Issue by asekawa** [wso2/product-is#17003]
**Describe the issue:**

When trying the SSO flow , with console and my account having two language modes , the UI provides mix language.
<img width="1512" alt="Screenshot 2023-10-17 at 12 29 02" src="https://github.com/wso2/product-is/assets/40081059/1de662ec-628a-45ec-a977-54bc24ec25a8">

**How to reproduce:**

1.Login to my account
2.Set Sinhala as the language mode
3.SSO to console (Make sure the language is set to English from the sign in page)

**Expected behavior:**

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 - Product Version: [e.g., IS 5.10.0, IS 5.9.0]
 - OS: [e.g., Windows, Linux, Mac]
 - Database: [e.g., MySQL, H2]
 - Userstore: [e.g., LDAP, JDBC]

---

### Optional Fields

**Related issues:**

**Suggested labels:**

**Comment by asekawa** [comment]
More pages which contains this issue

<img width="1041" alt="Screenshot 2023-10-17 at 13 50 31" src="https://github.com/wso2/product-is/assets/40081059/a8b64926-237e-4bc7-a1e4-5b9f3e99ac36">
<img width="1058" alt="Screenshot 2023-10-17 at 13 50 18" src="https://github.com/wso2/product-is/assets/40081059/4e2e41e6-ebef-4393-8c25-758152336f67">

**Comment by aaujayasena** [comment]
@brionmario  @UdeshAthukorala is this expected?

**Comment by brionmario** [comment]
> @brionmario @UdeshAthukorala is this expected?

No. We should disable `i18n` in Console. Console currently doesn't fully support i18n and is not configured to read the server language preference cookie value like My Account.

**Comment by pavinduLakshan** [comment]
Re-opening as framework configurations are not reflected in the console deployed in the IS pack.

**Comment by pavinduLakshan** [comment]
Verified the fix in the latest build.

https://github.com/wso2/product-is/assets/41533942/ab06e732-80cb-428b-9515-c6bdc884f0e3
